load_easybuild_module.sh

```diff
@@ -75,17 +75,22 @@ else
     if [[ $? -eq 0 ]]; then
         echo_green ">> Module for EasyBuild v${EB_VERSION} found!"
     else
-        eb_ec=EasyBuild-${EB_VERSION}.eb
-        echo_yellow ">> Still no module for EasyBuild v${EB_VERSION}, trying with easyconfig ${eb_ec}..."
-        ${EB} --search ${eb_ec} | grep ${eb_ec} > /dev/null
+        module --ignore_cache avail 2>&1 | grep -i easybuild/${EB_VERSION} &> ${ml_av_easybuild_out}
         if [[ $? -eq 0 ]]; then
-            echo "Easyconfig ${eb_ec} found for EasyBuild v${EB_VERSION}, so installing it..."
-            ok_msg="EasyBuild v${EB_VERSION} installed, alright!"
-            fail_msg="Installing EasyBuild v${EB_VERSION}, yikes! (output: ${eb_install_out})"
-            ${EB} EasyBuild-${EB_VERSION}.eb 2>&1 | tee -a ${eb_install_out}
-            check_exit_code $? "${ok_msg}" "${fail_msg}"
+            echo_green ">> Module for EasyBuild v${EB_VERSION} found!"
```

Comment:
You'll need to add another line:

IGNORE_CACHE='--ignore_cache'  # Module load should also ignore the Lmod cache

And then below (line 111 I believe:

```
module ${IGNORE_CACHE} load EasyBuild/${EB_VERSION}
```

Reply:
I added the line too early, so moved it down.